docker/detect_rRNA/parse_tRNA.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
```

Comment:
Same comments as "parse_rRNA-bacteria.py"

Reformatted a bit:
```python
#!/usr/bin/env python

# This file is part of MGnify genome analysis pipeline.
#
# MGnify genome analysis pipeline is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.

# MGnify genome analysis pipeline is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.

# You should have received a copy of the GNU General Public License
# along with MGnify genome analysis pipeline. If not, see <https://www.gnu.org/licenses/>.

import sys
import os
import argparse

aa = [
    "Ala",
    "Gly",
    "Pro",
    "Thr",
    "Val",
    "Ser",
    "Arg",
    "Leu",
    "Phe",
    "Asn",
    "Lys",
    "Asp",
    "Glu",
    "His",
    "Gln",
    "Ile",
    "Tyr",
    "Cys",
    "Trp",
]

if __name__ == "__main__":
    parser = argparse.ArgumentParser(description="Parse tRNA")
    parser.add_argument(
        "-i", "--input", dest="input", required=True, help="File with the tRNA"
    )
    args = parser.parse_args()
    input_file = args

    with open(input_file, "r") as f:
        trnas = 0
        for line in f:
            if not "Isotype / Anticodon" in line:
                continue
            cols = line.split()
            if len(cols) > 1:
                aa_pred = line.split(":")[0].split()[0]
                counts = int(line.split(":")[1].split()[0])
                if (aa_pred in aa or "Met" in aa_pred) and counts > 0:
                    trnas += 1

    clean_file_name = os.path.basename(input_file).split("_stats")[0]

    print("{name}\t{trnas}".format(name=clean_file_name, trnas=trnas))

```